unkeyed/unkey
# refactor: use biome imports sorting

## What does this PR do?



As Unkey continues to receive an increasing number of contributions, introducing further standardization to our codebase is essential for enhancing maintainability. Given that Biome is already used as formatter/linter, adopting their [Imports Sorting](https://biomejs.dev/analyzer/#imports-sorting) feature would significantly improve the organization of our import declarations. This step not only aligns with the current toolset but also promotes coding consistency and readability across the project.

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

Closes issue #955



## Type of change




## How should this be tested?



- Test A: Change some import declaration and then run `pnpm fmt`. The expected output is that the imports should be sorted according to Biome's [rules](https://biomejs.dev/analyzer/#how-imports-are-sorted).

## Checklist



### Required


### Appreciated
